app/components/courses/international_students_component.rb

```diff
@@ -0,0 +1,21 @@
+module Courses
+  class InternationalStudentsComponent < ViewComponent::Base
+    include ViewHelper
+
+    def initialize(provider:)
+      @provider = provider
+    end
+
+    def visa_sponsorship_status
+      if @provider.can_sponsor_all_visas?
+        'can sponsor Student and Skilled Worker visas, but this is not guaranteed'
+      elsif @provider.can_only_sponsor_student_visa?
+        'can sponsor Student visas, but this is not guaranteed'
+      elsif @provider.can_only_sponsor_skilled_worker_visa?
+        'can sponsor Skilled Worker visas, but this is not guaranteed'
+      else
+        'cannot sponsor visas'
+      end
```

Comment:
This content is slightly out-of-date from the [prototype](https://find-prototype.herokuapp.com), which uses this wording (and includes a link):

"We are unable to sponsor a visa. You will need to [get the right visa or status to study in the UK](https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-non-uk-applicants#visas-and-immigration)."

or

"We can [sponsor Student visas](https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-non-uk-applicants#visas-and-immigration), but this is not guaranteed."

Also, the logic of which to display need to take the course type (fee-paying or salaried) into account, ie:

```
if course == salaried && provider_can_sponsor_skilled_worker_visa
  // can sponsor Skilled worker visa
elsif course == fee_paying && provider_can_sponsor_student_visa
  // can sponsor Skilled worker visa
else
  // cannot sponsor visa
endif
```

Reply:
Thats updated now. The new logic means that we never show _We can sponsor Student and Skilled Worker visas_. I think it needs an update to publish as well to keep things in sync...